blazorbootstrap/Components/Toasts/Toast.razor.cs

```diff
@@ -17,6 +17,8 @@ public partial class Toast : BaseComponent, IDisposable
         protected override void BuildClasses(ClassBuilder builder)
         {
             builder.Append(BootstrapClassProvider.Toast());
+            builder.Append($"text-{BootstrapClassProvider.ToToastTextColor(ToastMessage.Type)}");
+            builder.Append($"bg-{BootstrapClassProvider.ToToastBackgroundColor(ToastMessage.Type)}");
```

Comment:
@Shecla Please exclude this code. Current simple toast background color, and text color needs to be updated in `BootstraClassProvider`. Please create an issue. I'll do an enhancement.

![image](https://user-images.githubusercontent.com/2337067/191962782-21776ccb-b688-4207-8a51-684115e28821.png)